lib/reflifc/include/reflifc/Module.h

```diff
@@ -14,8 +15,18 @@ namespace reflifc
         {
         }
 
-        const char* owner() const { return ifc_->get_string(module_reference_->owner); }
-        const char* partition() const { return ifc_->get_string(module_reference_->partition); }
+        const char* owner() const {
```

Comment:
These 2 methods look very similar :). Let's introduce method `const char* get_string_optional(ifc::TextOffset) const` to avoid copypaste.